contracts/dao-core/src/msg.rs

```diff
@@ -231,6 +231,64 @@ pub enum QueryMsg {
 
 #[cw_serde]
 pub enum MigrateMsg {
-    FromV1 { dao_uri: Option<String> },
+    FromV1 {
+        dao_uri: Option<String>,
+        params: Option<Box<MigrateParams>>,
```

Comment:
did you add the `Box` here to stop clippy from complaining about variant sizes? I think its OK for these two varaints to be differently sized.

feel free to disable the lint: https://github.com/rust-lang/rust-clippy#allowingdenying-lints